packages/atomic/src/components/search/facets/atomic-automatic-facet/atomic-automatic-facet.tsx

```diff
@@ -0,0 +1,150 @@
+import {isNullOrUndefined} from '@coveo/bueno';
+import {AutomaticFacet, SearchStatus, FacetValue} from '@coveo/headless';
+import {Component, Prop, State, h, VNode} from '@stencil/core';
+import {
+  FocusTarget,
+  FocusTargetController,
+} from '../../../../utils/accessibility-utils';
+import {getFieldValueCaption} from '../../../../utils/field-utils';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {FacetContainer} from '../../../common/facets/facet-container/facet-container';
+import {FacetHeader} from '../../../common/facets/facet-header/facet-header';
+import {FacetValueCheckbox} from '../../../common/facets/facet-value-checkbox/facet-value-checkbox';
+import {FacetValueLabelHighlight} from '../../../common/facets/facet-value-label-highlight/facet-value-label-highlight';
+import {FacetValuesGroup} from '../../../common/facets/facet-values-group/facet-values-group';
+import {Hidden} from '../../../common/hidden';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+
+/**
+ * An automatic facet is a facet generated by the automatic facets feature. This component should
+ * not be used directly. It is internally used by the facet manager to automatically render updated
+ * automatic facets based on the search results.
+ *
+ * @part facet - The wrapper for the entire facet.
+ *
+ * @part label-button - The button that displays the label and allows to expand/collapse the facet.
+ * @part label-button-icon - The label button icon.
+ * @part clear-button - The button that resets the actively selected facet values.
```

Comment:
Seeing as facet value exclusion is in the works, we should perhaps keep this more generic. This way, we don't need to re-document things if/when we support facet value exclusion for this component 🙂 
```suggestion
 * @part clear-button - The button that resets all facet values to their idle state.
```

Reply:
same thing here.